initiatives/submit-a-request/index.html

```diff
@@ -0,0 +1,433 @@
+
+            <div class="form-actions form-actions__right-on-med ">
+                <input class="btn form-actions_item"
+                       type="submit"
+                       value="Generate FOIA PDF">
+                <a href="#" class="btn btn__link form-actions_item">
+                    Sample Letter
+                    <span class="cf-icon cf-icon-pdf"></span>
+                </a>
+            </div>
+
+        </form>
+    </section>
+    <!-- End Form Module. -->
+    <section class="block">
+        <h3>How to submit</h3>
+        <p>
+            To minimize delays associated with mail delivery
+            or other possible misrouting of your request,
+            we recommend submitting your request by email.
+            However, we can also accept by fax or mail.
+        </p>
+        <div class="content-l content-l__main">
+            <div class="content-l_col
+                        content-l_col-1-3
+                        block
+                        block__flush-top">
+                <h5>
+                    <span class="cf-icon cf-icon-email"></span>
+                    Email
+                </h5>
+
+                <p class="short-desc">
+                    <a href="mailto:FOIA@cfpb.gov">FOIA@cfpb.gov</a>
+                </p>
+            </div>
+            <div class="content-l_col
+                        content-l_col-1-3
+                        block
+                        block__flush-top">
+                <h5>
+                    <span class="cf-icon cf-icon-mail"></span>
+                    Mail
+                </h5>
+                <p class="short-desc">
+                    Consumer Financial Protection Bureau<br/>
+                    Attention: Chief FOIA Officer<br/>
+                    1700 G Street NW<br/>
+                    Washington, DC 20552
+                </p>
+            </div>
+            <div class="content-l_col
+                        content-l_col-1-3
+                        block
+                        block__flush-top">
+                <h5>
+                    <span class="cf-icon cf-icon-fax"></span>
+                    Fax
+                </h5>
+                <p class="short-desc">
+                    (855) 237-2392
+                </p>
+            </div>
+        </div>
+        <h3>Write your own letter</h3>
+        <p>
+            If you are writing your own letter,
+            make certain to include, at a minimum,
+            the information below.
+            Our sample letter can provide more guidance.
+        </p>
+        <p>A request should include:</p>
+        <ul>
+            <li>Your contact information
+                (e.g., full name, address, phone number, email address).</li>
+            <li>As many details as possible about the information
+                you are seeking from the Consumer Financial Protection Bureau
+                (e.g., “Consumer complaints received between January 1, 2013
```

Comment:
I don't think comma is necessary for a list of two items, nor is it in the approved copy I got. Pinging @jameshupp to confirm.